specs/v1.yaml

```diff
@@ -1651,6 +1875,57 @@ components:
                   This is useful for nonprofits to indicate when they have received and processed the grant.
                   Expressed in ISO 8601 format.
                   A null value indicates that the grant has not been acknowledged.
+    RecurringGrantCaptureRequest:
+      description: |-
+        The request to create and submit a monthly recurring grant. This is useful to capture a recurring grant intent associated with Connect workflow session.
+        The request should specify the grant amount. The grant will re-occur on the same day of the month that the grant was submitted.
```

Comment:
i would make the description regarding recurrence frequency more generic since monthly is a parameter of the request and we may change or add feature later on that allows difference recurring schedule. saying something like: "The grant will re-occur according to the scheduled frequency or recurrence of the request."